FrontPage/controllers/AddTaskViewController.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+class AddTaskViewController: UIViewController {
+  
+  
+  @IBOutlet weak var nameField: UITextField!
```

Comment:
It may be best to rename `nameField` to `titleField`. Title is used in the graphql calls.

Reply:
good point